README.md

```diff
@@ -108,6 +108,9 @@ Process check based on HTTP query.
                             Health check name.
       -g PROCESS_GROUP, --process-group PROCESS_GROUP
                             Supervisor process group name.
+      -pn PROCESS_NAME, --process-name PROCESS_NAME
```

Comment:
Can you please make it capital P (-P) ? I don't like having multiple characters for a short option.

Reply:
specifically for the http_check file or for all checks? There's a conflict in http_check and xmlrpc_check where -P is taken up by password. Any other flag suggestion?

Reply:
Yeah, it should be consistent for all checks... Maybe use -n or -N then (stands for process_Name)?

Reply:
Updated to -N